test/src/org/labkey/test/tests/cds/CDSGroupTest.java

```diff
@@ -183,6 +189,40 @@ public void verifyGroups()
         cds.deleteGroupFromSummaryPage(STUDY_GROUP);
 
         cds.clearFilters();
+
+        //Compose new Group
+        cds.goToSummary();
+        cds.clickBy("Studies");
+        cds.selectBars(CDSHelper.STUDIES[0], CDSHelper.STUDIES[1]);
+        cds.useSelectionAsSubjectFilter();
+
+        CDSHelper.NavigationLink.PLOT.makeNavigationSelection(this);
+
+        XAxisVariableSelector xaxis = new XAxisVariableSelector(this);
+        YAxisVariableSelector yaxis = new YAxisVariableSelector(this);
+
+        xaxis.openSelectorWindow();
+        xaxis.pickSource(CDSHelper.TIME_POINTS);
+        xaxis.pickVariable(CDSHelper.TIME_POINTS_DAYS);
+        xaxis.confirmSelection();
+        sleep(CDSHelper.CDS_WAIT_ANIMATION);
+        yaxis.pickSource(CDSHelper.NAB);
+        yaxis.pickVariable(CDSHelper.NAB_TITERIC50);
+        yaxis.confirmSelection();
+
+        cds.saveGroup(GROUP_PLOT_TEST , "");
```

Comment:
saveGroup method was intended to be used for groups created from Find Subject page only. The modification to applyAndMaybeWaitForBars resulted in a dozen test failures so I've reverted it.